TESTING.md

```diff
@@ -66,4 +69,32 @@ https://app.codecov.io/gh/opensearch-project/REPO-NAME-HERE/settings/badge
 1. Add a `codecov.yml` file into the repository ([example](https://github.com/opensearch-project/k-NN/commit/f7d1985230ce851cb97a7e41d8bce32127a4f33b))
 2. Set the target code coverage properly in case it blocks your PR in Github checks.
 
-See Codecov docs to learn more about [codecov yaml file](https://docs.codecov.com/docs/codecov-yaml) and [common configurations](https://docs.codecov.com/docs/common-recipe-list).
\ No newline at end of file
+See Codecov docs to learn more about [codecov yaml file](https://docs.codecov.com/docs/codecov-yaml) and [common configurations](https://docs.codecov.com/docs/common-recipe-list).
+
+### Backwards Compatibility Testing
+
+The purpose of backwards compatibiity tests in plugins is to ensure that the plugin is safely upgraded when an OpenSearch node undergoes an upgrade. Backwards compatibility tests can be implemented in plugins by extending the OpenSearch bwc framework. The tests and the framework have been added in [`anomaly-detection`](https://github.com/opensearch-project/anomaly-detection) plugin which can be used as an example to add tests to other plugins.
+
+The changes required in the OpenSearch bwc framework to support plugins bwc tests have been added as part of [opensearch#1051](https://github.com/opensearch-project/OpenSearch/pull/1051)
```

Comment:
I would say `See [OpenSearch#1051](https://github.com/opensearch-project/OpenSearch/pull/1051) for more information.`